src/Symfony/Bundle/FrameworkBundle/Tests/Command/YamlLintCommandTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command;
+
+use Symfony\Component\Yaml\Tests\Command\LintCommandTest as BaseTest;
+use Symfony\Bundle\FrameworkBundle\Command\YamlLintCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * Tests the YamlLintCommand.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class YamlLintCommandTest extends BaseTest
+{
+    /**
+     * @return CommandTester
+     */
+    protected function createCommandTester()
+    {
+        $command = new YamlLintCommand();
+        $application = new Application();
+        $application->add($command);
```

Comment:
`$command` becomes (hopefully) itself line 34. How about `$application->add(new YamlLintCommand())` ? Wouldn't that be less confusing ?